src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- The maximum permitted value of the `COCKROACH_RPC_INITIAL_WINDOW_SIZE` environment variable has been increased to `64MB`. By increasing this environment variable, in conjunction with tuning OS-level maximum TCP window size, you can increase the throughput that [Raft replication](https://www.cockroachlabs.com/docs/v23.1/architecture/replication-layer#raft) can sustain over high latency network links. [#111287][#111287]
+- Updated Go version to 1.19.13. [#112204][#112204]
+
+<h3 id="v23-1-12-sql-language-changes">SQL language changes</h3>
+
+- New datetime built-ins (`make_date`, `make_timestamp`, and `make_timestamptz`) have been added, allowing for the creation of timestamps, timestamps with time zones, and dates. In addition, `date_trunc` now allows for a timestamp to be truncated in a specified timezone (to a specified precision). [#110338][#110338]
+- New system privileges `CREATEROLE`, `CREATELOGIN`, `CREATEDB`, and `CONTROLJOB` have been introduced. Each is analogous to its existing [role option](https://www.cockroachlabs.com/docs/v23.1/create-user#role-options) counterpart but can additionally be inherited through role membership. [#110359][#110359], [#110359][#110359], [#110359][#110359], [#110359][#110359]
+- [`RESTORE`](https://www.cockroachlabs.com/docs/v23.1/restore) can now be passed a `WITH EXECUTION LOCALITY` option similar to `BACKUP` to restrict execution of the job to nodes with matching localities. [#110611][#110611]
+- The `statement_activity` and `transaction_activity` table column `execution_total_cluster_seconds` is now accurate. The `combinedstmts` endpoint returns the correct value for the `StmtsTotalRuntimeSecs` and `TxnsTotalRuntimeSecs` properties. [#109639][#109639]
+- SQL commands that were previously limited to the `admin` [system privilege](https://www.cockroachlabs.com/docs/v23.1/security-reference/authorization#supported-privileges) can now be used by users with the `VIEWCLUSTERMETADATA` or `REPAIRCLUSTERMETADATA` system privilege, depending on whether the operation is read-only or modifies state. [#111058][#111058]
```

Comment:
I'm unsure if this should be SQL or Security. It seems like it's duplicated above.